src/linked-list.js

```diff
@@ -1,9 +1,41 @@
+class Node {
+  constructor(element) {
+    this.value = element;
+    this.next = null;
+  }
+}
+
 class LinkedList {
   constructor() {
     this.head = null;
     this.tail = null;
     // Do not modify anything inside of the constructor
   }
+  addToTail(element) {
+    const newNode = new Node(element);
+    if (this.head === null) {
```

Comment:
You can use the `!` operator here instead of `this.head === null`. It is more concise and covers more cases, including if this.head is undefined instead of just null.